samples/Actor/DemoActor/DemoActor.cs

```diff
@@ -120,11 +134,15 @@ protected override Task OnDeactivateAsync()
         /// </summary>
         /// <param name="data">Timer input data.</param>
         /// <returns>A task that represents the asynchronous operation.</returns>
-        private Task TimerCallBack(object data)
+        public Task TimerCallBack(object data)
```

Comment:
nitpick: spelling - usually this would be `Callback` instead of `CallBack` because it's a single word when used as a programming term